python/semantic_kernel/connectors/ai/chat_completion_client_base.py

```diff
@@ -7,10 +7,14 @@
 from semantic_kernel.services.ai_service_client_base import AIServiceClientBase
 
 if TYPE_CHECKING:
-    from semantic_kernel.connectors.ai.prompt_execution_settings import PromptExecutionSettings
+    from semantic_kernel.connectors.ai.prompt_execution_settings import (
```

Comment:
A small note: whatever is formatting these changes doesn't need to until one runs the pre-commit, which does run `ruff` and will format if/when necessary.

Reply:
My VSCode has auto formatting enabled on save. I do still have the pre-commit hook setup so I hope my changes still follow the intended format.